src/engine.cc

```diff
@@ -85,6 +85,7 @@ void EngineController::PopulateOptions(OptionsParser* options) {
                             "time-curve-left-width") = 82.0f;
   options->Add<FloatOption>(kTimeCurveRightWidth, 0.0f, 1000.0f,
                             "time-curve-right-width") = 74.0f;
+  options->Add<BoolOption>("Ponder", "ponder") = false;
```

Comment:
Maybe add a comment that we actually don't use it, but GUIs require that to exist?

Reply:
Will do. Actually, I don't even know which GUIs, if any, require that option to exist, but the UCI spec has the comment: "this means that the engine is able to ponder." next to the option without making it really clear whether that comment is meant for the engine (ie. if the option is true, the engine knows it is able to ponder) or the GUI (ie. if the option exists, the GUI knows the engine is able to ponder). I thought it was safer to offer the option for GUIs that chose the second interpretation.